demos/websocket_demo.cc

```diff
@@ -37,8 +37,22 @@ int main(int argc, char** argv) {
     app.run(argc, argv, [] () -> seastar::future<> {
         return async([] {
             websocket::server ws;
-            ws.register_handler("echo", [] (temporary_buffer<char> &&buf, output_stream<char> &write_buf) {
-                return write_buf.write(std::move(buf));
+            ws.register_handler("echo", [] (input_stream<char>& in,
```

Comment:
Don't you need to squash this patch with the previous patch, which used this in a test?

Reply:
We have decided to separate the commits which update the demo and the test, even though they use the same 'echo' handler.
I hope it is not in contradiction with the general style.

Reply:
it's not a contradiction, demos are tests are separate entities, it's fine